pyproject.toml

```diff
@@ -44,6 +44,7 @@ pre-commit = "*"
 black = "22.3.0"
 ruff = "*"
 tox = "*"
+pylint = "*"
```

Comment:
we stopped using `pylint` by the way, since it stopped working properly with `pydantic` 2.*. Something maybe you could also look at if you have time.